### ISSUE TITLE
Makefile:492: recipe for target 'libnetmaumaucommon.la' failed

### ISSUE THREAD

**Issue by Takios** [velnias75/NetMauMau#32]
Hello,

makeing fails with the error in the title. I've saved the whole make output in the following gist:
https://gist.github.com/Takios/15e6596326eecb236938

Greetings
Takios

**Comment by velnias75** [comment]
@Takios can you provide me more information?

* configure command (`./config.status -V`)
* which libtool version
* gcc version
* distribution
* etc

Did you build from a clone or the released tarball? If you updated from a existing repo, do a make clean before.

I'm wondering: the file missing is the first compiled at the beginning of the output. So why got it lost?

**Comment by Takios** [comment]
Hey,

tried to build it again today after a reboot beginning from a clean state, cloned again etc. and it worked without issues. Maybe some side effects from an update yesterday or maybe I missed something. 

At first, I thought that using automake-1.15 (1.11 cannot be easily obtained on this system) was the culprit, but 1.15 seems to be working fine.

Greetings

**Comment by velnias75** [comment]
Meanwhile every version of automake should work. I gradually changed the Makefiles and I could also remove that note from `README.md`

The only problem I sometimes encounter in any projects are, that changes in libraries sometimes don't cause a relink. In this cases just `make clean` helps. If `config.h` changed `make -k && make` (ignore all errors first, than try again) often solves the build problem.

I don't know where less disk space messages appear on your system, but maybe that was the reason.

But eventually I'm happy you got it built :smile_cat:

**Comment by velnias75** [comment]
@Takios BTW: are you build RPM's? I would be interested in them for test builds on OBS.

**Comment by Takios** [comment]
Haven't built an RPM before so I wanted to get into it using a small project.
It's primarily for learning purposes so it could take a while before I can deliver a working RPM. ^^

**Comment by velnias75** [comment]
Small project? If you want to make matching packages NetMauMau will result in 
5 (6 with dbg) packages.


Long time ago I did RPM's, but meanwhile every RPM-based distro requires its 
own tweaks, so I decided not to offer RPMs by myself.

I do the Debian packaging (a good pointer for you as well. Look into the 
`debian` directory, as far as I know for RPM you won't need that several 
files, but only one SPEC file) and Gentoo. For the latter one I have a nice 
mentor who also helped me kickstarting into Git.

Other distros I try every now and then on OBS if I find a build file as base.

Wish you luck :)
